homeassistant/components/deconz/light.py

```diff
@@ -170,3 +172,33 @@ def device_state_attributes(self):
             attributes['all_on'] = self._device.all_on
 
         return attributes
+
+
+class DeconzGroup(DeconzLight):
+    """Representation of a deCONZ group."""
+
+    def __init__(self, device, gateway):
+        """Set up group and create an unique id."""
+        super().__init__(device, gateway)
+
+        self._unique_id = '{}-{}'.format(
+            self.gateway.api.config.bridgeid,
+            self._device.deconz_id)
+
+    @property
+    def unique_id(self):
+        """Return a unique identifier for this device."""
+        return self._unique_id
+
+    @property
+    def device_info(self):
+        """Return a device description for device registry."""
+        bridgeid = self.gateway.api.config.bridgeid
+
+        return {
+            'connections': {(CONNECTION_ZIGBEE, self.unique_id)},
```

Comment:
I believe `connections` is meant to describe links to other devices, right?  What does adding this do?

Reply:
A bit too quick there, yeah it isn't necessary